src/content/getting-started/quickstart/external/_index.md

```diff
@@ -75,6 +75,24 @@ This guide assumes cluster-a's kubeconfig file is named `kubeconfig.cluster-a` a
 subctl deploy-broker --kubeconfig kubeconfig.cluster-a --globalnet
 ```
 
+#### Label gateway nodes
+
+When Submariner joins a cluster to the broker via the `subctl join` command, it chooses a node on which to install the 
+gateway. By default, Submariner uses a worker node, but if there are no worker nodes, it does not install a gateway 
+unless a node is explicitly labeled as gateway. Since we are deploying k3s all-in-one nodes, there are no worker nodes, 
+so it is necessary to label the single node as gateway. By default, the node-name is the hostname. In this example, the 
```

Comment:
```suggestion
When Submariner joins a cluster to the broker via the `subctl join` command, it chooses a node on which to install the 
gateway by labeling it appropriately. By default, Submariner uses a worker node but, if there are no worker nodes, then no gateway is installed 
unless a node is manually labeled as gateway. Since we are deploying k3s all-in-one nodes, there are no worker nodes, 
so it is necessary to label the single node as gateway. By default, the node name is the hostname. In this example, the 
```

Reply:
Thanks for the review!  I pushed a change with your suggestions plus a couple of grammatical tweaks.